silo/tests/test_views.py

```diff
@@ -855,3 +855,50 @@ def test_no_merge_name(self, mock_merged_silos_map, mock_merge_two_silos):
         self.assertEqual(response.url, '/silo_detail/{}/'.format(silo.id))
         self.assertIn(left_read, silo.reads.all())
         self.assertIn(right_read, silo.reads.all())
+
+
+class OneDriveViewsTest(TestCase):
+    fixtures = ['../fixtures/read_types.json']
+
+    def setUp(self):
+        self.org = factories.Organization()
+        self.tola_user = factories.TolaUser(organization=self.org)
+        self.user = factories.User()
+
+    def test_onedrive_js_page(self):
+        self.client.force_login(self.user)
+        response = self.client.get('/onedrive')
+
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'silo/onedrive.html')
+
+    def test_onedrive_js_page_no_login(self):
+        response = self.client.get('/onedrive')
+
+        self.assertEqual(response.status_code, 302)
+        self.assertTemplateNotUsed(response, 'silo/onedrive.html')
```

Comment:
Which template should be used?

Reply:
oh, sorry. This is when a user is not logged in and is redirected. So i think it is not using any template but redirecting to API. That's why i only check that this template should not be used.

Reply:
ok. Thanks :smiley: